src/main/java/me/partlysanestudios/partlysaneskies/dungeons/RequiredSecretsFound.java

```diff
@@ -62,8 +62,6 @@ public void checkRequiredSecrets(TickEvent.ClientTickEvent event) {
 
                 alreadySendThisRun = true;
                 break;
-            } else if (line.contains("Secrets Found")){
```

Comment:
Kinda sad this had to go, but for some reason it just stopped working

Anyway the performance is still very good